### PR TITLE
Set opt template

### PR DESCRIPTION
**Pull request by prestonfromspotx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,6 @@ liblifthttp - The fast asynchronous C++17 HTTP client library
 
 You're using curl? Do you even lift?
 
-Copyright (c) 2017, Josh Baldwin
-
 https://github.com/jbaldwin/liblifthttp
```
Comment (tcashel) [inline README.md]: still have jbaldwin's repo url

Comment (tcashel) [inline README.md]: still have jbaldwin's repo github url

Comment (jbaldwin) [reply README.md]: this is my private version, I'm thinking you meant to push this to the spotx one?

**Comment by tcashel** [comment]
My bad Josh

**Comment by jbaldwin** [comment]
Closing due to wrong repo

**Pull request closed without merging**